### PR TITLE
Add Profile RayyanRNKJ

### LINKED ISSUE #837: Rayyan Rasheed
CapturePage(4)

### PR DESCRIPTION
**Pull request by RayyanRNKJ**
resolve #837

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1582,5 +1582,11 @@
...
-}
-]
+  },
+  {
+    "name": "Rayyan Rasheed",
+    "githubUsername": "RayyanRNKJ",
+    "imageUrl": "https://user-images.githubusercontent.com/64679235/141696204-3084d6cf-49d0-452a-979e-35a58b89b4dc.png",
+    "issueNumber": 837
+  }
+]
```
Comment (krishguptadev) [inline data.json:1592]:
```suggestion
]

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please fix what @krishguptadev  suggested

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**